docs/excel/custom-functions-errors.md

```diff
@@ -30,12 +82,13 @@ function getComment(commentID) {
       return json.body;
     })
     .catch(function (error) {
-      throw error;
+      throw new CustomFunctions.Error(CustomFunctions.ErrorCode.notAvailable);
     })
 }
 ```
 
 ## Next steps
+
 Learn how to [troubleshoot problems with your custom functions](custom-functions-troubleshooting.md).
 
 ## See also
```

Comment:
You may want to make sure debugging article has a See also link as well to this article for discoverability purposes